docrepr/webview.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*
+
+"""
+View object documentation HTML in a separate GUI window.
+
+Requires PyQt4 or PySide. Original code by
+anatoly techtonik <techtonik@gmail.com> is placed in
+public domain.
```

Comment:
This is code that could be copy/pasted into different project that need a webview for their HTML stuff without reference to Spyder, and me is one of those people person. You may remove the note yourself, but I submit it as a public domain contribution. And you can remove the note, because it is public domain, but I don't want to do this myself for the reason stated - if I need the code later, I will copy/paste the code from my commit that is clearly marked as PD.

It is a common practice to take public domain code and modify it.
https://github.com/python/cpython/blob/1fe0fd9feb6a4472a9a1b186502eb9c0b2366326/Modules/_ctypes/libffi/install-sh
https://github.com/python/cpython/blob/c1e0265342dbfc710ecdd4be6fab707f7bd74dce/Lib/doctest.py

So I don't get what are you objecting of. It is not about me trying to get more credit than other people - it is just about making clear about origin of the code.